Makefile

```
@@ -8,6 +8,7 @@ version = 1.20231001.0
 .PHONY: version
 
 version:
+	@echo $(version) > elixir/VERSION
```

```suggestion
	@echo $(version) > VERSION
```

Would it make sense to use this in the top-level so other components don't redefine it?

---

```suggestion
	@echo -n $(version) > elixir/VERSION
```

Could strip the newline and prevent having to `trim/1` later.

---

I did `String.trim/1` on purpose because some editors always insert a newline at the end of file, so it's safer to always trim it anyways.

---

@jamilbk 

> Would it make sense to use this in the top-level so other components don't redefine it?

I don't think it will play nicely with docker builds because that file will be always out of build context.